packages/babel-preset-env/test/fixtures/bugfixes/v8-spread-parameters-in-optional-chaining-chrome-89-no-bugfixes/stdout.txt

```diff
@@ -8,15 +8,15 @@ Using targets:
 Using modules transform: auto
 
 Using plugins:
-  syntax-numeric-separator { "chrome":"89" }
-  syntax-nullish-coalescing-operator { "chrome":"89" }
-  proposal-optional-chaining { "chrome":"89" }
-  syntax-json-strings { "chrome":"89" }
-  syntax-optional-catch-binding { "chrome":"89" }
-  syntax-async-generators { "chrome":"89" }
-  syntax-object-rest-spread { "chrome":"89" }
-  transform-modules-commonjs { "chrome":"89" }
-  proposal-dynamic-import { "chrome":"89" }
-  proposal-export-namespace-from {}
+  syntax-numeric-separator
+  syntax-nullish-coalescing-operator
+  proposal-optional-chaining { chrome }
```

Comment:
small thing, but I guess this is confusing without looking at the data, that it's because you need to add bugfixes? Seems like it implies chrome doesn't support it.

Guess it'll sorta be fixed when we make it default?

Reply:
Yes, no version of chrome supports optional chaining unless you enable bugfixes.